opentech/apply/dashboard/views.py

```diff
@@ -25,42 +25,41 @@
     PaymentRequestsDashboardTable,
     ProjectsDashboardTable
 )
-from opentech.apply.utils.views import ViewDispatcher
+from opentech.apply.utils.views import RedirectToListViewMixin, ViewDispatcher
 
 
-class AdminDashboardView(TemplateView):
-
-    def get(self, request, *args, **kwargs):
-        # redirect to submissions list when we use the filter to search for something
-        if len(request.GET):
-            query_str = '?'
-            for key, value in request.GET.items():
-                query_str += key + '=' + value + '&'
-            return HttpResponseRedirect(reverse_lazy('funds:submissions:list') + query_str)
+class AdminDashboardView(RedirectToListViewMixin, TemplateView):
+    destination_lut = {
```

Comment:
not a fan of `lut` usage - can we be a bit more descriptive about this variable.

Reply:
What would you like it to be?